code/src/components/Submit.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+export const Submit = ({setSummary}) => {
+
+  const formSubmit = event => {
+    event.preventDefault()
+    setSummary(true)
```

Comment:
Currently, one can still press submit even if not filled in a single input, perhaps you should implement a way to not allow the button the send to the summary until all inputs are filled in